computed_stats.go

```diff
@@ -0,0 +1,45 @@
+package server
+
+import (
+	"net/http"
+	"strconv"
+)
+
+func (s *Server) ListMetadataByUserID(w http.ResponseWriter, r *http.Request) {
+	// TO DO: get userID from authentication to use it here instead of "1"
+	stats, err := s.ComputedStatsService.ListMetadataByUserID(1)
+	if err != nil {
+		writeError(w, ErrNotFound.Wrap(err)) // TODO differentiate not found and decoding
+		return
+	}
+
+	writeJSON(w, stats, 200)
+}
+
+func (s *Server) FindComputedStatsByID(w http.ResponseWriter, r *http.Request) {
+	idInt, err := getStringParamAndConvertToInt(r, idParam)
```

Comment:
Actually the `ComputedStats` ID must be a string, because it matches the ID of the `Report` stored in Firestore (which is a string). Sorry for not spotting that earlier! 😕